src/renderer/components/theme-settings/theme-settings.js

```diff
@@ -39,6 +39,7 @@ export default defineComponent({
         'black',
         // Second group
         'nordic',
+        'gruvboxDark',
```

Comment:
Please insert this in the third group below `dracula`
https://github.com/FreeTubeApp/FreeTube/pull/4805/files#diff-a0480aa13261014e61fefded30097805ea7559cfe7b068b66b209ae896b17adaR47
```suggestion
```